misc/assets/anonymization.svg

```diff
@@ -0,0 +1,47 @@
+<svg version="1.1" viewBox="0 0 1000 200" xmlns="http://www.w3.org/2000/svg">
+  <style>
+      text {
+        font-family: sans-serif;
+      }
+      .caption {
+        font-size: 12pt;
+        text-anchor: middle;
+        dominant-baseline: central;
+      }
+      .note {
+        font-size: 9pt;
+        text-anchor: middle;
+        dominant-baseline: ideographic;
+      }
+      .relation {
+        fill: none;
+        stroke: black;
+        stroke-width: 1pt;
+      }
+      .relation.from-many {
+        marker-start: url(#relation-many)
+      }
+  </style>
+
+  <defs>
+    <marker id="relation-many" orient="auto" markerWidth="6" markerHeight="6" refX="0" refY="3">
+      <circle cx="3" cy="3" r="3" />
+    </marker>
+    <g id="model">
+      <rect class="box" x="-125" y="-20" width="250" height="40" fill="none" stroke="black" stroke-width="1pt" />
```

Comment:
Templaty v defs s rovnakym ID by vo vsetkych nasich SVG mali byt totozne. Takze ak tu nieco ma ID `model`, tak vo vsetkych dalsich suboroch vsetko s ID `model` by mali byt totozne s tymto templatom. Inac budeme mat v tych templatoch chaos.

Pripadne (ak to funguje) [skus vydelit `defs`] do samostatneho suboru. Ale neskusal som, ci to bude fungovat.

[skus vydelit `defs`]: https://stackoverflow.com/questions/20783374/i-it-possible-to-include-external-svg-defs

Ak niekde potebujeme rozne typy pre `model` tak to treba spravit bud dvomi templatemi napr. `model` a `model-wide`. Alebo este lepsie cez classy ak sa da `<use href="#model" class="wide">` a  nastylovat to nejak csskom.